controllers/activemqartemis_reconciler.go

```diff
@@ -1514,15 +1549,24 @@ func MakeVolumes(customResource *brokerv1beta1.ActiveMQArtemis, namer Namers) []
 		if "" != connector.SSLSecret {
 			secretName = connector.SSLSecret
 		}
-		volume := volumes.MakeVolume(secretName)
-		volumeDefinitions = append(volumeDefinitions, volume)
+		if _, ok := secretVolumes[secretName]; !ok {
+			volume := volumes.MakeVolume(secretName)
+			volumeDefinitions = append(volumeDefinitions, volume)
+			secretVolumes[secretName] = secretName
+		}
 	}
 
 	if customResource.Spec.Console.SSLEnabled {
 		clog.V(1).Info("Make volumes for ssl console exposure on k8s")
 		secretName := namer.SecretsConsoleNameBuilder.Name()
-		volume := volumes.MakeVolume(secretName)
-		volumeDefinitions = append(volumeDefinitions, volume)
+		if "" != customResource.Spec.Console.SSLSecret {
+			secretName = customResource.Spec.Console.SSLSecret
+		}
```

Comment:
How is this related to #499?

Reply:
I believe this is a bug (it should have been there, otherwise the user specified secret will be ignored).